core/src/main/java/rdx/works/core/sargon/PersonasExtensions.kt

```diff
@@ -24,8 +24,8 @@ val Persona.factorSourceId: FactorSourceId
 val Persona.derivationPathScheme: DerivationPathScheme
     get() = securityState.derivationPathScheme
 
-val Persona.derivationPathEntityIndex: HDPathValue
-    get() = securityState.transactionSigningFactorInstance.publicKey.derivationPath.nonHardenedIndex
+val Persona.derivationPathEntityIndex: HdPathComponent
+    get() = securityState.transactionSigningFactorInstance.publicKey.derivationPath.path.components.last()
```

Comment:
`last()` should never throw since we don't allow empty derivation path component array.